### ISSUE TITLE
GPU pls?

### ISSUE THREAD

**Issue by filieski** [tiny-dnn/tiny-dnn#537]
Hi there,

First of all, I'm not sure that if it's the correct thing to label this as an issue but I'll give it a try anyways.
Even tho I enjoy this framework very much and I love the header-only idea I ran into somewhat of an issue. As an undergrad student I'm working on an image recognition application for android.
This idea is this:
1. convert AlexNet caffe model
2. include tiny-dnn in qt and show predictions of the model visually
3. ?????
4. PROFIT!

But there's very long delay (something around 30 seconds on computer and a over a minute on android). I should know better than to complain about open source software, but take this merely as a suggestion or idea. It'd be awesome if there was GPU support in tiny-dnn.

TL;DR: Is GPU support for android (libgles maybe) in tiny-dnn coming soon?

Regards,

**Comment by bhack** [comment]
For mobile we are also waiting for quantization and I suggest you also to find a more compact model. See https://github.com/tiny-dnn/tiny-dnn/issues/202 @wangyida

**Comment by smissan** [comment]
I second the request - libDNN backend appears to be broken now - does not compile.

**Comment by bhack** [comment]
See https://github.com/tiny-dnn/tiny-dnn/issues/286#issuecomment-249204555 /cc @edgarriba
